massgov/openmass
# DP-23017: Document how to debug twig templates

**Description:**
Added documentation to the openmass project for Drupal Twig debugging.


**Jira:** (Skip unless you are MA staff)
DP-23017


**To Test:**
- Verify the documentation is clear and understandable.
- Verify the information is accurate through testing.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
